packages/design-system/src/components/OcModal/OcModal.vue

```diff
@@ -347,6 +350,14 @@ export default defineComponent({
       required: false,
       default: false
     },
+    /**
+     * Password policy for the input
+     */
+    inputPasswordPolicy: {
+      type: Object,
+      required: false,
+      default: () => {}
```

Comment:
I guess you want to return an empty object, no? Currently it's just an empty function.

```suggestion
      default: () => ({})
```